benchalerts/benchalerts/conbench_dataclasses.py

```diff
@@ -53,6 +53,20 @@ class RunComparisonInfo:
     compare_results: Optional[List[dict]] = None
     benchmark_results: Optional[List[dict]] = None
 
+    @staticmethod
+    def _result_display_name_from_compare_dict(result: dict) -> str:
+        """Generate a name for a benchmark result to be displayed in a notification list.
+
+        Given: the compare endpoint's payload's "contender" key dict:
+        https://conbench.ursa.dev/api/redoc#tag/Comparisons/paths/~1api~1compare~1benchmark-results~1%7Bcompare_ids%7D~1/get
+        """
+        display_name = f'`{result["benchmark_name"]}`'
+        if result["language"] and result["language"] != "unknown":
+            display_name += f' ({result["language"]})'
+        if result["case_permutation"] != "no-permutations":
+            display_name += f' with {result["case_permutation"]}'
+        return display_name
```

Comment:
:+1:  I am sure that in the future there will be more work towards displaying the "main identity" of a benchmark result via human-oriented text even more consistently across the product surface (including docs/glossary).